doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,134 @@ Release management
-* 10 meaning LTS.
-
-So, each MAJOR release series goes through a development-maturity life cycle of
-MINOR releases, as follows:
-
-1. **Alpha**. Once in every few months we release a few alpha versions,
-   e.g. 2.0.1, 2.0.2.
-
-   Alpha versions may contain incompatible changes, crashes and other bugs.
-
-2. **Beta**. Once major changes necessary to introduce new flagship features
-   are ready, we release a few beta versions, e.g. 2.1.3, 2.1.4.
-
-   Beta versions may contain crashes, but do not have incompatible changes,
-   so can be used to develop new applications.
-
-4. **Stable**. Finally, after we see our beta versions run successfully in
-   production, usually in a few more months, during which we fix all incoming
-   bugs and add some minor features, we declare this MAJOR release series
-   stable.
-
-Like Ubuntu, we distinguish two kinds of stable releases:
-
-* **LTS (Long Term Support)** releases that are supported for 3 years
-  (community) and up to 5 years (paying customers). **LTS** release
-  is identified by MINOR version 10.
 * **Standard stable releases** are only supported a few months after the next
   stable is out.
 
 "Support" means that we continue fixing bugs in a release.
 
-We add commits simultaneously to three MAJOR releases:
+We add bug fixes simultaneously into the following three releases and
+into the current alpha release:
 
-* **LTS** is a stable release which does not receive new features, and only gets
-  backward compatible fixes. Hence, following the rules of semver, LTS release
+* **LTS** is a stable release which does not receive new features and only gets
+  backward compatible fixes. Hence, LTS release
   never has its MAJOR or MINOR version increased, and only gets PATCH level
   releases.
 
-* **STABLE** is our current stable release, which may receive new features.
+* **STABLE** are our current stable releases. They also receive no new features
+  and only backward compatible fixes.
   When the next STABLE version is published, MINOR version is incremented.
-  Between MINOR releases, we may have intermediate PATCH level releases as well,
-  which will contain only bug fixes. We maintain PATCH level releases for
-  two STABLE releases, the current and the previous one, to preserve support
-  continuity.
-
-* **NEXT** is our next MAJOR release, and it follows the maturity
-  cycle described in the beginning. While NEXT release is in alpha state,
-  its MINOR is frozen at 0 and is only increased when the release reaches
-  BETA status. Once the NEXT release becomes STABLE, we switch the vehicle for
-  delivery of minor features, designating the previous stable release as LTS,
-  and releasing it with MINOR set to 10.
+  We maintain PATCH level releases for two STABLE releases,
+  the current and the previous one, to preserve support continuity.
+
+* **NEXT** Beta is our next MINOR release, and it follows the maturity cycle
+  described in the beginning.
```

Comment:
Is there any reason to introduce the second term 'next' with the same meaning as 'beta'? I generally don't like this: it is better to have one term for one thing. This way it is easier to understand.